includes/wikia/tests/FixDomainNameTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @group FixDomainName
+ */
+class FixDomainNameTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider fixDomainNameDataProvider
+	 */
+	public function testFixDomainName( $env, $name, $language, $expected ) {
+		$this->mockEnvironment($env);
+		$this->assertEquals( $expected, Wikia::fixDomainName( $name, $language ) );
+	}
+
+	public function fixDomainNameDataProvider() {
+		return [
+			[
+				'env' => WIKIA_ENV_PROD,
+				'name' => 'foo',
+				'language' => 'en',
+				'execpted' => 'foo.wikia.com'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "execpted" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)